src/v5_hal/firmware/include/NodeManager.h

```diff
@@ -57,12 +59,15 @@ class NodeManager {
 // the superclass constructor (should look like :Node([manager], [time]))
 class Node {
 protected:
-    ros::NodeHandle* m_handle;
-
+    

```

Comment:
This space + protected keyword can be removed

Reply:
Resolved & removed